src/main/java/io/lighty/yang/validator/formats/Analyzer.java

```diff
@@ -58,9 +58,13 @@ private static boolean submodulesAreNotEmpty(final Collection<? extends ModuleLi
     @SuppressFBWarnings(value = "SLF4J_SIGN_ONLY_FORMAT",
                         justification = "Valid output from LYV is dependent on Logback output")
     private void printOut() {
+        if (testedModule != null) {
```

Comment:
Attribute `testedModule` is always null in Analyzer format, so this part can be removed.

Reply:
this commit was removed, since this issue will not be fixed.